packages/table/src/useLeafygreenTable/useLeafygreenTable.types.ts

```diff
@@ -2,57 +2,48 @@ import { RefObject } from 'react';
 import { VirtualItem } from 'react-virtual';
 import { Cell, Row, Table, TableOptions } from '@tanstack/react-table';
 
-// Below is copied from react-virtual as their types are exported
+import { VirtualizerValues } from './reactVirtual.types';
 
-type ScrollAlignment = 'start' | 'center' | 'end' | 'auto';
-
-interface ScrollToOptions {
-  align: ScrollAlignment;
-}
-
-interface ScrollToOffsetOptions extends ScrollToOptions {}
-interface ScrollToIndexOptions extends ScrollToOptions {}
-
-export interface VirtualizerValues {
-  virtualItems: Array<VirtualItem>;
-  totalSize: number;
-  scrollToOffset: (index: number, options?: ScrollToOffsetOptions) => void;
-  scrollToIndex: (index: number, options?: ScrollToIndexOptions) => void;
-  measure: () => void;
-}
-
-// Above is copied from react-virtual as their types are not exported
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type LGRowData = {
+  [key: string]: any;
+};
 
-export type LeafygreenTableType<T extends unknown> = T & {
-  renderExpandedContent?: (row: Row<T>) => JSX.Element;
+export type LGTableDataType<T extends LGRowData> = T & {
+  renderExpandedContent?: (row: LeafygreenTableRow<T>) => JSX.Element;
 };
 
-export type LeafygreenTableCell<T extends unknown> = unknown &
-  Cell<LeafygreenTableType<T>, unknown>;
+export type LeafygreenTableCell<T extends LGRowData> = Cell<
+  LGTableDataType<T>,
+  LGRowData
+>;
 
-export type LeafygreenTableRow<T extends unknown> = unknown &
-  Row<LeafygreenTableType<T>>;
+export interface LeafygreenTableRow<T extends LGRowData>
+  extends Row<LGTableDataType<T>> {}
```

Comment:
Everything feels great in this PR, except I may go back and replace all usages of the `Leafygreen` prefix like in `LeafygreenTableRow` with `LG` following your pattern